pypeit/scripts/sensfunc.py

```diff
@@ -66,17 +66,6 @@ def get_parser(cls, width=None):
                                  'provided but with .fits trimmed off if it is in the filename.')
         parser.add_argument("-s", "--sens_file", type=str,
                             help='Configuration file with sensitivity function parameters')
-        parser.add_argument("-f", "--flatfile", type=str,
-                            help="R|Use a flat calibration file to compute the blaze function when generating "
-                                 "the sensitivity function.  This is helpful to account for small scale undulations "
-                                 "in the sensitivity function.  Note that it is not possible to set --flatfile and "
-                                 "simultaneously use a .sens file with the --sens_file option. If "
-                                 "you are using a .sens file, set the flatfile there via e.g.:\n\n"
-                                 "F|    [sensfunc]\n"
-                                 "F|         flatfile = Calibrations/Flat_A_0_DET01.fits\n\n"
-                                 "Where Flat_A_0_DET01.fits is the flat file in your "
-                                 "Calibrations directory\n")
-
```

Comment:
To maintain functionality, do we need to add a parameter that allows the user to not use the extracted flat-field spectrum during the sensitivity calculation?